apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -1,11 +1,123 @@
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  width: 100%;
+  background-color: #D32E6A;
+  padding: 0.5em;
+  cursor: pointer;
+}
+
+.close-btn {
+  position: absolute;
+  top: -6.4rem;
+  right: 0;
+  color: #D32E6A;
```

Comment:
Color `#D32E6A` should be written as `#d32e6a`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexnotation)